sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -29,8 +29,18 @@ namespace Azure.Messaging.EventHubs.Samples
     ///      dotnet Azure.Messaging.EventHubs.Samples.dll `
     ///          --FullyQualifiedNamespace "{yournamespace}.servicebus.windows.net" `
     ///          --EventHub "<< YOUR_EVENT_HUB_NAME >>" `
-    ///          --TenantId " << YOUR_TENANT_ID >> " `
-    ///          --ClientId "<< YOUR_CLIENT_ID >>" `
```

Comment:
The arguments are the names of the properties. The properties are Tenant and Client. Trying to use TenantId and ClientId as arguments do not function.

Reply:
Thanks.   Looks like we missed an update during one of the refactors.